Xamarin.Forms.Platform.Android/AppCompat/PickerRenderer.cs

```diff
@@ -141,6 +141,9 @@ void UpdatePicker()
 		{
 			Control.Hint = Element.Title;
 
+			if (Element.TitleColor != default(Color))
```

Comment:
here and in other places replace please on `Element.IsSet(Picker.TitleColorProperty)`

Reply:
Done